.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
🤔

Reply:
I don't know how it got there, but I thought you would know, since you committed it  :D